### PR TITLE
Button Animations

### PR DESCRIPTION
**Pull request by mmmann12**

### CODE REVIEW

File: src/home/HomeButtons.js [hunk 1]
```diff
@@ -41,4 +41,113 @@ export const FlexItem = styled(Button)`
...
+  border-radius: 10px;
+  width: 400px;
+  height: 100px;
+  border: none;
+  color: #fff;
+  text-align: center;
+  font-size: 20px;
+  transition: all 0.5s;
+  cursor: pointer;
+  cursor: pointer;
```
Comment (yusefmustafa) [inline src/home/HomeButtons.js:96]: duplicate

File: src/home/HomeButtons.js [hunk 2]
```diff
@@ -41,4 +41,113 @@ export const FlexItem = styled(Button)`
...
+  padding-left:8px;
+  }
+  :hover:after {
+  opacity: 1;
+  right: 115px;
+  }
+`
+
+export const ButtonStart = styled(Button)`
+  border-radius: 4px;
```
Comment (yusefmustafa) [inline src/home/HomeButtons.js:83]: this is redefined on line 87

File: src/home/HomeButtons.js (hunk 1 above)
Comment (mmmann12) [reply src/home/HomeButtons.js]: fixed

File: src/home/HomeButtons.js (hunk 2 above)
Comment (mmmann12) [reply src/home/HomeButtons.js]: fixed

File: src/home/HomeButtons.js (hunk 1 above)
Comment (yusefmustafa) [reply src/home/HomeButtons.js]: looks like all the buttons have this issue

File: src/home/HomeButtons.js (hunk 2 above)
Comment (yusefmustafa) [reply src/home/HomeButtons.js]: all the buttons have this issue, not just `ButtonStart `

Comment (mmmann12) [reply src/home/HomeButtons.js]: yeah changed all buttons

File: src/home/HomeButtons.js (hunk 1 above)
Comment (mmmann12) [reply src/home/HomeButtons.js]: done

**Review by yusefmustafa — APPROVED** [review]

**Pull request merged**